src/components/MenuButton/MenuButton.md

```diff
@@ -0,0 +1,76 @@
+### Notes for use
+
+Context menu should be used as a menu button where you want to provide options that are hidden behind a click.
+
+### Examples
+
+MenuButton with default Icon, and all applicable children properties
+
+```js { "props": { "data-description": "with deafult icon and children properties" } }
+const Reply = require('../Icon/icons/Reply').default;
+const Edit = require('../Icon/icons/Edit').default;
+const Share = require('../Icon/icons/Share').default;
+
+<div>
+  <MenuButton
+    ariaLabel="aria"
+    menuItems={[{
+      key: 'regular',
+      icon: Edit,
+      text: 'Icon Item',
+    }, {
+      key: 'noIcon',
+      text: 'No Icon',
+      onClick: () => console.log('no icon'),
+    }, {
+      key: 'disabledIcon',
+      text: 'Icon and disabled',
+      icon: Reply,
+      isDisabled: true,
+      onClick: () => console.log('icon and disabled'),
+    }, {
+      key: 'header',
+      text: 'Header Type with Divider After',
+      type: 2,
+    }, {
+      key: 'divider',
+      type: 1,
```

Comment:
are we unable to use the enum here in the .md file instead of the compiled int value?